src/main/java/cinema/project/service/MovieSessionServiceImpl.java

```diff
@@ -25,4 +25,19 @@ public MovieSessionServiceImpl(MovieSessionDao movieSessionDao) {
     public MovieSession add(MovieSession session) {
         return movieSessionDao.add(session);
     }
+
+    @Override
+    public MovieSession update(MovieSession movieSession) {
+        return movieSessionDao.update(movieSession);
+    }
+
+    @Override
+    public boolean remove(MovieSession movieSession) {
+        return movieSessionDao.remove(movieSession);
+    }
```

Comment:
```suggestion
    @Override
    public void remove(MovieSession movieSession) {
        return movieSessionDao.remove(movieSession);
    }
```
in your controller you don't return anything